opentelemetry-api/tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
```

Comment:
Curious, given we will have other packages (e.g. opentelemetry-sdk) under this repo, do we want to have a single global tox, or a tox file per package?

Reply:
I don't know what best practises are here, but I'd imagine we will have quite some code/config duplication with multiple toxfiles 🤔 I think OpenCensus-python used [nox](https://nox.thea.codes/en/stable/), maybe that tool is better in this regard?

Reply:
From a conversation with @reyang: nox gives us a combined coverage report if we have branches that depend on python version, but tox made it easier to run tests in parallel on travis. We'll stick with tox and burn the coverage bridge when we come to it.

I don't know whether it'll be easier to have one or multiple tox files. It does make it easier now to have it under `opentelemetry-api` since there's no code yet to check in the SDK.